packages/api/test/e2e/rxapi.create.e2e.ts

```diff
@@ -32,6 +34,26 @@ describe('e2e rx api create', () => {
     });
   });
 
+  it ('Should create rx api instance with slim metadata', async done => {
+    const provider = config.wsProvider[`${process.env.TEST_TYPE}`];
+    const api = await Api.create({provider, fullMeta: false});
+    const stakingAssetId = await api.query.genericAsset.stakingAssetId();
+    expect(api.query.nft).toBeUndefined();
+    expect(stakingAssetId.toNumber()).toBeGreaterThan(0);
+    await api.disconnect();
+    done();
+  });
+
+  it ('Should create rx api instance with custom metadata', async done => {
+    const provider = config.wsProvider[`${process.env.TEST_TYPE}`];
+    const api = await Api.create({provider, modules:['TransactionPayment', 'GenericAsset']});
```

Comment:
when user specifies only module they are interested in... the mandatory module like System, TimeStamp would be auto included